README.md

```diff
@@ -6,6 +6,13 @@ Official bootstrap for running your own [Sentry](https://sentry.io/) with [Docke
 
  * Docker 1.10.0+
  * Compose 1.6.0+ _(optional)_
+ 
+ ## Minimum Hardware Requirements:
```

Comment:
I don't know what we'd actually recommend, but probably much more than these requirements. 5GB of disk won't go very far, and 1Ghz single core CPU is so slow. I'd probably be more comfortable if we just suggest minimum memory required.

Reply:
On my experience, 1Ghz single core CPU is just slow while migration beacuse CPU usage is 99%,
After installation,  CPU usage get back to normal.

Reply:
I host 5 projects on sentry with 1vCPU and 2GB of RAM and never had a problem. My RAM is normally around 60%. I also managed to run sentry on 1GB of RAM but accessing the UI shortly after a bug was reported got very slow and produced timeouts.